### PR TITLE
Garbage collection, Watchdog, Dragged obj's immune to pressure diffs

### PR DESCRIPTION
**Pull request by WhiteFox08**
HIGH HO SILVER

### CODE REVIEW

**Comment by Chronitonity** [comment]
Holy shit

File: code/controllers/Processes/inactivity.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 				if(!istype(C.mob, /mob/dead))
 					log_access("AFK: [key_name(C)]")
 					C << "<SPAN CLASS='warning'>You have been inactive for more than 10 minutes and have been disconnected.</SPAN>"
-					del(C)
+					qdel(C)
```
Comment (ghost) [inline code/controllers/Processes/inactivity.dm:13]: dont qdel clients, del clients.

File: code/controllers/configuration.dm [hunk 2]
```diff
@@ -159,7 +159,7 @@
 				probabilities[M.config_tag] = M.probability
 				if(M.votable)
 					votable_modes += M.config_tag
-		del(M)
+		qdel(M)
```
Comment (ghost) [inline code/controllers/configuration.dm:162]: don't qdel this, this happens before roundstart.

File: code/controllers/configuration.dm [hunk 3]
```diff
@@ -483,7 +483,7 @@
 		var/datum/game_mode/M = new T()
 		if(M.config_tag && M.config_tag == mode_name)
 			return M
-		del(M)
+		qdel(M)
```
Comment (ghost) [inline code/controllers/configuration.dm:486]: don't qdel this, this happens before roundstart.

File: code/controllers/configuration.dm [hunk 4]
```diff
@@ -492,10 +492,10 @@
 		var/datum/game_mode/M = new T()
 		//world << "DEBUG: [T], tag=[M.config_tag], prob=[probabilities[M.config_tag]]"
 		if(!(M.config_tag in modes))
-			del(M)
+			qdel(M)
```
Comment (ghost) [inline code/controllers/configuration.dm:495]: don't qdel this, this happens before roundstart.

File: code/controllers/configuration.dm [hunk 5]
```diff
@@ -492,10 +492,10 @@
 		var/datum/game_mode/M = new T()
 		//world << "DEBUG: [T], tag=[M.config_tag], prob=[probabilities[M.config_tag]]"
 		if(!(M.config_tag in modes))
-			del(M)
+			qdel(M)
 			continue
 		if(probabilities[M.config_tag]<=0)
-			del(M)
+			qdel(M)
```
Comment (ghost) [inline code/controllers/configuration.dm:498]: don't qdel this, this happens before roundstart.

**Comment by WhiteFox08** [comment]
@Chnkr are you sure? MSO reported less round start time.

**Comment by ghost** [comment]
@whitefox08 I'm pretty sure qdel is slower, and not needed before roundstart.

**Pull request merged**